### DISCUSSION TITLE
MeshLab 2022.02 is out!

### DISCUSSION THREAD

**Discussion by alemuntoni** [cnr-isti-vclab/meshlab#1218] [General]
This version brings lot of bugfixes, minor improvements and features, and a new filter plugin for computing parametrizations.

Along with this, we also released [PyMeshLab 2022.2](https://github.com/cnr-isti-vclab/PyMeshLab/discussions/187), which brings lot of improvements along with a renaming of all the filter function names, with the goal of making more clear what each filter does.

We thank  @alfonsosanchezbeato for making available his MeshLab Plugin for computing the minimal volume bounding box of a mesh. Extra plugins maintained by third parties can be downloaded from the [Release Page](https://github.com/cnr-isti-vclab/meshlab-extra-plugins/releases) of the [MeshLab Extra Plugins repository](https://github.com/cnr-isti-vclab/meshlab-extra-plugins), and they can be loaded on MeshLab by going to Help -> Plugin Info -> Load Plugins.

Changelog:

- new plugin for computing parametrizations (Harmonic and LSCM)
- various bugfixes and improvements
- possibility to select number of threads in filter screened poisson
- proper support of multitextured gltf files
- new revisited and deterministic filter searcher
- some minor new features on set texture and vertex displacement filters

**Comment by Takehito-Hayashi** [comment]
May I ask you a question?
Download of Meshlab 2022.2(WIN64)  dose not work.
(Files not found)
It only to do is download the source files and make?

**Reply by alemuntoni** [reply]
Sorry, there has been a small change a few days ago that broke the links. They should be ok now!

**Reply by Takehito-Hayashi** [reply]
Thank you very much for your reply.
And You are providing a excellent app. 'Meshlab' , thank you too.

**Reply by operationbegfordinner** [reply]
install error: 
![image](https://user-images.githubusercontent.com/101020201/156898680-32eb6733-f711-406c-8fff-9085c8dc06ae.png)

**Reply by alemuntoni** [reply]
Hi @operationbegfordinner, just installing the [Microsoft Visual Studio Redist](https://aka.ms/vs/17/release/vc_redist.x64.exe) should solve the issue. It has not been automatically packed in the meshlab installation due to recent updates of some dependencies we have in our build system.

**Comment by Dovahkiii** [comment]
Hello,

Is MeshLab 2022.02 is compatible with Windows 11 ?

And thank you for this great app :>

**Reply by alemuntoni** [reply]
Yes, it is!

**Reply by Dovahkiii** [reply]
Tip top, thanks a lot !

**Comment by mikefree41** [comment]
I downloaded MeshLab and installed and it won't run because MANY files seem to be missing where can I get a complete operating copy of MeshLab?

**Comment by myselfhimself** [comment]
Hello, is there such a thing a voxel remesh in Meshlab? Right now I use blender 3.x's same named modifier for this, but I wish I had some external tool for that.

**Comment by BigTsung** [comment]
Hi guys, 
I encounter a crash issue when importing a gltf file to meshlab2022.02. But if I import the same gltf file to meshlab 2021.10, it is work fine. Any clues or suggestions? Thanks

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/121002770/208350381-66b1e183-f941-4e18-a8ed-d7a07be93617.png)
can't instal
